hw_pandas/Pandas_plain_text.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pandas_profiling as pp
+
+train_df = pd.read_csv("https://raw.githubusercontent.com/Serfentum/bf_course/master/14.pandas/train.csv")
+
+sns.displot(train_df, x="A")
+sns.displot(train_df, x="T")
+sns.displot(train_df, x="G")
+sns.displot(train_df, x="C")
+
+
+for nucleotide in ['A', 'T', 'G', 'C']:
+    train_df[nucleotide].hist(bins=50, alpha=0.5, label=nucleotide)
+plt.legend(loc='upper right')
+plt.title('Histogram of A, T, G, C')
+plt.xlabel('number of contigs with this nucleotide')
+plt.ylabel('count')
+plt.show()
+
+train_part = train_df.loc[:,
+                          ['pos', 'reads_all',
+                           'mismatches', 'insertions']][train_df.mismatches > train_df.mismatches.mean()]

```

Comment:
Нужно было отобрать по колонке `matches`, хотя суть от этого не изменится. В `.loc` тоже можно передавать логический вектор, так что всё это можно немного упростить
```suggestion
train_part = train_df.loc[train_df.mismatches > train_df.mismatches.mean(), ['pos', 'reads_all', 'mismatches', 'insertions']]

```